### PR TITLE
Docs: Making it clear where helpers can be imported from

### PR DESCRIPTION
**Pull request by wesruv**
Didn't know where to get the helpers from, took me a while to track it down myself. I think this note, or something like it would be helpful.

### CODE REVIEW

**Comment by bennypowers** [comment]
those helpers are all available from `@open-wc/testing`, though?

**Comment by wesruv** [comment]
I'm not sure the intention in the last comment: if it's:
* that I should use a different file import, that's cool with me, my ask is that it's made clearer what to import
* if it's a question of if all those functions exist in the imported file, I'm pretty sure the answer is yes

File: docs/docs/testing/helpers.md [hunk 1]
```diff
@@ -152,6 +152,12 @@ await elementUpdated(el);
 expect(el.foo).to.equal('bar');
 ```
 
+All of these helpers must be imported from `@open-wc/testing/index-no-side-effects.js`, e.g.
+
+```js
+import { nextFrame, aTimeout, waitUntil } from '@open-wc/testing/index-no-side-effects.js';
```
Comment (bennypowers) [inline docs/docs/testing/helpers.md:158]:
```suggestion
import { nextFrame, aTimeout, waitUntil } from '@open-wc/testing';
```

**Review by bennypowers — APPROVED** [review]

File: docs/docs/testing/helpers.md (hunk 1 above)
Comment (Westbrook) [reply docs/docs/testing/helpers.md]: Should both package names be the same here? e.g. should "All of these helpers must be imported from `@open-wc/testing/index-no-side-effects.js`" be "All of these helpers must be imported from `@open-wc/testing`"?

File: docs/docs/testing/helpers.md [hunk 2]
```diff
@@ -152,6 +152,12 @@ await elementUpdated(el);
 expect(el.foo).to.equal('bar');
 ```
 
+All of these helpers must be imported from `@open-wc/testing/index-no-side-effects.js`, e.g.
```
Comment (bennypowers) [inline docs/docs/testing/helpers.md:155]:
```suggestion
All of these helpers must be imported from `@open-wc/testing`, e.g.
```

File: docs/docs/testing/helpers.md (hunk 1 above)
Comment (bennypowers) [reply docs/docs/testing/helpers.md]: yes

Comment (wesruv) [reply docs/docs/testing/helpers.md]: Oops, I resolved this when I meant to commit it. Now I can't commit the suggestion 😩

Comment (wesruv) [reply docs/docs/testing/helpers.md]: Oh, looks like it was committed, perfect.

**Review by Westbrook — APPROVED** [review]

**Comment by wesruv** [comment]
Thank you!

**Pull request merged**